### PR TITLE
PAAS-2022: fix default value for variable because of Cloudscripting

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-2022

Short description:

### CODE REVIEW

File: mixins/jahia.yml [hunk 1]
```diff
@@ -236,7 +236,8 @@ actions:
 
         # Enable APM by default
         echo "apm_config:" >> /etc/datadog-agent/datadog.yaml
-        echo "  enabled: ${DATADOG_APM_ENABLED:-true}" >> /etc/datadog-agent/datadog.yaml
+        [ -z ${DATADOG_APM_ENABLED} ] && DATADOG_APM_ENABLED=true
+        echo "  enabled: ${DATADOG_APM_ENABLED}" >> /etc/datadog-agent/datadog.yaml
```
Comment (laurentfufu) [inline mixins/jahia.yml:240]: the main datadog conf file is handle by `set_dd_tags.sh` cronified  script and it's already put this line with `true` as default value (cf [here](https://github.com/Jahia/jahiastic-jahia/blob/master/base/files/set_dd_tags.sh#L13-L14))

File: mixins/jcustomer.yml [hunk 2]
```diff
@@ -350,4 +350,5 @@ actions:
     - cmd [cp]: |-
         setenv=$(find /opt/jcustomer/jcustomer/bin -name setenv)
         echo 'APM_OPTS="-javaagent:'${JCUSTOMER_HOME}'/jcustomer/datadog/dd-java-agent.jar -Ddd.logs.injection=true -Ddd.service=jcustomer -Ddd.env='${env.envName}' -Ddd.profiling.enabled=true"' >> $setenv
-        echo '${DATADOG_APM_ENABLED:-true} && JAVA_OPTS+=" $APM_OPTS"' >> $setenv
+        echo '[ -z ${DATADOG_APM_ENABLED} ] && DATADOG_APM_ENABLED=true' >> $setenv
+        echo '${DATADOG_APM_ENABLED} && JAVA_OPTS+=" $APM_OPTS"' >> $setenv
```
Comment (laurentfufu) [inline mixins/jcustomer.yml:354]:
```suggestion
        # this fucking cloudscripting interprets the bash form `${<variable name>:<something>}` as its own placeholder
        # so we have to trick it
        echo '${DATADOG_APM_ENABLED¤-true} && JAVA_OPTS+=" $APM_OPTS"' | sed 's/¤/:/' >> $setenv
```

I found this way more readable and has the advantage of explaining why a side trip is necessary here

File: mixins/jahia.yml (hunk 1 above)
Comment (adoux) [reply mixins/jahia.yml]: I removed this part

File: mixins/jcustomer.yml (hunk 2 above)
Comment (adoux) [reply mixins/jcustomer.yml]: Done, also done in the migration package

**Review by amitc-jahia — APPROVED** [review]

**Pull request merged**